deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,146 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    **Important Note**: This operator is purely for prototyping purposes and not
+    **Important Note**: 
+    * This operator is purely for prototyping purposes and not
     supported by any upstream community at this time.
 
-
-    **Important Note**: Currently, while in preview, this operator does not
+    * Currently, while in preview, this operator does not
     support automatic upgrades. You must remove the old version of the operator
     manually before installing a new version.
 
 
-    ## Using KubeFed
-
-    This deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion.
-    For example, if you subscribe to this package in namespace `my-namespace`,
-    KubeFed will be deployed to manage objects _only_ in `my-namespace` in
-    the host cluster and target clusters.
-
-    After installing KubeFed instance, you will use the `kubefedctl` command
-    line tool to do two types of configuration:
-      
-    - Configure federation to connect to your target clusters with `kubefedctl join`
-    - Create new federation APIs for the API types you want to federate with `kubefedctl enable`
-
-
-    **Important Note**: To add federation capability to your namespace, you are
-    only required to install kubefed in the hosting cluster; there is no need
-    to install kubefed in the target clusters (and no effect gained from
-    doing so).
-
-
-    **Important Note**: The instructions in this guide refer to the namespace
-    where federation is installed as `<namespace>` in command examples. The
-    upstream user guide uses `kube-federation-system` as the name of this namespace.
+    ## Start using the operator
 
-    ### Get the kubefedctl CLI tool
-
-    Before you use kubefed, you should ensure that you're using the binary
-    for this version. You can get it from the releases on GitHub.
+    This operator supports deploying a KubeFed control plane in both
+    namespace-scoped and cluster-scoped modes. For example, if you subscribe to
+    this package in a namespace `my-namespace`, KubeFed will be deployed to
+    manage objects _only_ in `my-namespace` in the host cluster and target
+    clusters.
+    
+    The operator provides the following CRDs for deploying components of a `KubeFed` control plane in a given cluster:
+    * KubeFedWebHook
+    * KubeFed
+    
+    ### KubeFed deployment
+    Follow these steps to deploy a KubeFed control plane:
+
+    * Install the operator by following the instructions in the
+    [documentation](https://docs.openshift.com/container-platform/4.1/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster).
+    The KubeFed operator only needs to be installed in the host cluster for the
+    deployment, not each individual target cluster.
```

Comment:
Change to "The KubeFed operator only needs to be installed in the host cluster for the deployment. It does not need to be installed in each target cluster."